src/mlpack/methods/gmm/gmm.hpp

```diff
@@ -160,18 +161,34 @@ class GMM
    * Return the probability that the given observation came from this
    * distribution.
    *
-   * @param observation Observation to evaluate the probability of.
+   * @param observation Observation vector to evaluate the probability.
    */
   double Probability(const arma::vec& observation) const;
 
+  /**
+   * Return the probability of the given observation (x) matrix.
+   *
+   * @param x Observation matrix.
+   * @param probs Vector to store probability value of observation x.
+   */
+  void Probability(const arma::mat& x, arma::vec& probs) const;
+
   /**
    * Return the log probability that the given observation came from this
    * distribution.
    *
-   * @param observation Observation to evaluate the probability of.
+   * @param observation Observation to evaluate the probability.
```

Comment:
Actually the `of` was correct here, but the sentence is a bit awkward anyway.  Maybe `for` makes more sense?